### PR TITLE
OEL-1863: Added file assert for PersonContentRenderTest.

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

**Review by AaronGilMartinez — DISMISSED** [review]

**Review by AaronGilMartinez — DISMISSED** [review]

File: tests/src/Functional/PersonContentRenderTest.php [hunk 1]
```diff
@@ -143,13 +144,30 @@ public function testPersonPage(): void {
...
-      'Person document test',
-      $files->filter('p')->text()
-    );
+    $files = $content->filter('.bcl-file-container');
 
+    $document_file = File::create([
+      'uri' => $this->getTestFiles('text')[0]->uri,
+    ]);
+    $document_file->save();
+    $absolute_file_url = $document_file->createFileUrl(FALSE);
```
Comment (drishu) [inline tests/src/Functional/PersonContentRenderTest.php]: absolute_file_url is not needed, you can call $document_file->createFileUrl(FALSE); directly in $expected_document

File: tests/src/Functional/PersonContentRenderTest.php [hunk 2]
```diff
@@ -143,13 +144,30 @@ public function testPersonPage(): void {
...
+        'icon' => 'file-text-fill',
+        'url' => $absolute_file_url,
+      ],
+      'translations' => NULL,
+      'link_label' => 'Download',
+    ];
     $this->assertCount(3, $files);
+    $assert = new FilePatternAssert();
+    foreach ($files as $file) {
+      $file_html = $file->ownerDocument->saveHTML($file);
```
Comment (drishu) [inline tests/src/Functional/PersonContentRenderTest.php]: $file should be a Crawler object, if not, use ->eq(), and then you can use $file->html() to get the html

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/src/Functional/PersonContentRenderTest.php (hunk 2 above)
Comment (tibi2303) [reply tests/src/Functional/PersonContentRenderTest.php]: done. thanks

File: tests/src/Functional/PersonContentRenderTest.php (hunk 1 above)
Comment (tibi2303) [reply tests/src/Functional/PersonContentRenderTest.php]: done. thanks

**Review by drishu — DISMISSED** [review]

**Review by AaronGilMartinez — DISMISSED** [review]

**Review by AaronGilMartinez — APPROVED** [review]

**Review by drishu — APPROVED** [review]

**Pull request merged**